basic/src/test/java/org/jboss/hal/testsuite/test/configuration/elytron/ElytronAuthenticationTestCase.java

```diff
@@ -0,0 +1,468 @@
+
+    @Test
+    public void removeHttpAuthenticationTest() throws Exception {
+        final String factoryName = RandomStringUtils.randomAlphanumeric(5),
+                authenticationName = RandomStringUtils.randomAlphanumeric(5);
+        final Address factoryAddress = elyOps.getElytronAddress(SERVICE_LOADER_HTTP_SERVER_MECHANISM_FACTORY,
+                factoryName),
+                authenticationAddress = elyOps.getElytronAddress(HTTP_AUTHENTICATION_FACTORY, authenticationName);
+        final ResourceVerifier authenticationVerifier = new ResourceVerifier(authenticationAddress, client);
+
+        try {
+            ops.add(factoryAddress, Values.of(MODULE, MODULE_NAME_1));
+            new ResourceVerifier(factoryAddress, client).verifyExists();
+            ops.add(authenticationAddress,
+                    Values.of(HTTP_SERVER_MECHANISM_FACTORY, factoryName).and(SECURITY_DOMAIN, SECURITY_DOMAIN_NAME));
+            authenticationVerifier.verifyExists();
+
+            page.navigate2application().selectFactory(HTTP_AUTHENTICATION_LABEL);
+
+            page.getResourceManager().removeResource(authenticationName).confirmAndDismissReloadRequiredMessage()
+                    .assertClosed();
+            assertFalse("Deleted authentication should not be present in the table any more!",
+                    page.resourceIsPresentInMainTable(authenticationName));
+            authenticationVerifier.verifyDoesNotExist();
+
+        } finally {
+            ops.removeIfExists(authenticationAddress);
+            ops.removeIfExists(factoryAddress);
+            adminOps.reloadIfRequired();
+        }
+    }
+
+    @Test
+    public void editHttpAuthenticationAttributesTest() throws Exception {
+        final String factoryName1 = RandomStringUtils.randomAlphanumeric(5),
+                factoryName2 = RandomStringUtils.randomAlphanumeric(5),
+                authenticationName = RandomStringUtils.randomAlphanumeric(5),
+                realm2name = RandomStringUtils.randomAlphanumeric(5),
+                domain2name = RandomStringUtils.randomAlphanumeric(5);
+        final Address factoryAddress1 = elyOps.getElytronAddress(SERVICE_LOADER_HTTP_SERVER_MECHANISM_FACTORY,
+                factoryName1),
+                factoryAddress2 = elyOps.getElytronAddress(SERVICE_LOADER_HTTP_SERVER_MECHANISM_FACTORY, factoryName2),
+                securityRealmAddress2 = createSecurityRealm(realm2name),
+                securityDomainAddress2 = createSecurityDomain(domain2name, realm2name),
+                authenticationAddress = elyOps.getElytronAddress(HTTP_AUTHENTICATION_FACTORY, authenticationName);
+        final ResourceVerifier authenticationVerifier = new ResourceVerifier(authenticationAddress, client);
+
+        try {
+            ops.add(factoryAddress1, Values.of(MODULE, MODULE_NAME_1));
+            ops.add(factoryAddress2, Values.of(MODULE, MODULE_NAME_2));
+            new ResourceVerifier(factoryAddress1, client).verifyExists();
```

Comment:
I changed my mind, you are right.  `#assertSuccess()` is probably sufficient here. Should I change it everywhere?